handlers/analytics.go

```diff
@@ -0,0 +1,130 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/julienschmidt/httprouter"
+	"github.com/livepeer/catalyst-api/errors"
+	"github.com/mmcloughlin/geohash"
+	"github.com/xeipuuv/gojsonschema"
+	"io"
+	"net/http"
+	"strconv"
+)
+
+const GEO_HASH_PRECISION = 3
+
+type AnalyticsHandler struct {
```

Comment:
Is this struct used anywhere or will we need it in a further PR?

Reply:
leftover, removed.